example/intercept/intercept.cc

```diff
@@ -538,9 +538,9 @@ TSPluginInit(int /* argc */, const char * /* argv */ [])
 {
   TSPluginRegistrationInfo info;
 
-  info.plugin_name   = (char *)PLUGIN;
-  info.vendor_name   = (char *)"MyCompany";
-  info.support_email = (char *)"ts-api-support@MyCompany.com";
+  info.plugin_name   = PLUGIN;
+  info.vendor_name   = "MyCompany";
+  info.support_email = "ts-api-support@MyCompany.com";
```

Comment:
I fixed `example/add-header` and `example/append-transform` this way. See if that looks reasonable. For `.cc` files I would replace `static` with the anonymous namespace.